tsconfig.json

```diff
@@ -13,5 +13,5 @@
     /* Skip type checking all .d.ts files. */
     "jsx": "react"
   },
-  "exclude": ["node_modules","jest.config.js"]
+  "exclude": ["node_modules","jest.config.js", "/Users/shivaylamba/Desktop/example-ts/__mocks__"]
```

Comment:
nice addition here 👍 i wonder if there is some way to make this a variable so we dont have to exclude this, otherwise i believe we will have to add everyones path here for running tests - no? @shivay-at-pieces

Reply:
I have updated this to include any directories including mock, so this should work for others as well.